### PR TITLE
missing keys

### PR DESCRIPTION
**Pull request by JaskRendix**
It started #466 with a dozen, now we are at more than 60 missing keys. 
This is everything missing, no exceptions.
I share as draft, so if there is any advice, etc. about the names, feel free.

### CODE REVIEW

File: mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po [hunk 1]
```diff
@@ -2405,6 +2405,198 @@ msgstr "Team Xero Grunt IX"
...
+msgid "taba_greeter"
+msgstr "Radio"
+
+msgid "tallon"
+msgstr "Tallon"
+
+msgid "tuxemart"
+msgstr "Tuxemart"
+
+msgid "xerogrunt1"
```
Comment (kerizane) [inline mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po]: There should already be a group of xerogrunts

Comment (JaskRendix) [reply mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po]: You gave me an idea

**Comment by JaskRendix** [comment]
It was a mess. NPCs spread everywhere, now everything is below the right label and most important sorted in alphabetical order.

**Comment by kerizane** [comment]
This diff is massive and difficult to read. What exactly did you do?

I would prefer the diff to be focused on adding the names.

**Comment by JaskRendix** [comment]
NPCs where split in two sections and they weren't in alphabetical order. I just reordered.

As you wish. I'll update.

**Comment by kerizane** [comment]
It might be good to order them in another ticket after this is merged.

**Comment by JaskRendix** [comment]
Sure

**Comment by kerizane** [comment]
If you could add your new names in alphabetical order, but  not reorder the existing ones, that would be good.

**Comment by JaskRendix** [comment]
Sure, as it was in the beginning. I'll move only the knights and xerogrunt as you pointed out before.

**Comment by JaskRendix** [comment]
Done, tell me if it's ok.

**Comment by kerizane** [comment]
@Sanglorian @Qiangong2 what do you think?

I'm wondering if all of these are actually necessary. When do we use the names, is it just for battle?

Do we want to define names for everyone, or does this just lead to extra work for translators?

**Comment by JaskRendix** [comment]
I don't think all these are necessary. Some are part of unused maps, but at the end the JSON exists. This shows the problem of recreating the same NPC again and again. Think about mom and homemaker, as well as postboy, we have already a spyder_postboy.

**Comment by Qiangong2** [comment]
All of the spyder npcs are for Sanglorian's campaign, which is why they look like duplicates.

The NPC names are in there for localization purposes.

**Comment by Sanglorian** [comment]
I believe names are shown when battle begins only. The NPCs without any name given are ones I use for NPCs on the overland only - so spyder_picnicker might stand in for multiple picnickers on different maps.

**Comment by kerizane** [comment]
If they're only used for battle then I don't think we should be giving them names for no reason. @Sanglorian @Qiangong2 do you agree?

I'm planning to write some map validation logic at some point, we can check for missing names there.

Sorry for the wasted effort @JaskRendix

**Comment by JaskRendix** [comment]
No problem, remember to close the #466

**Pull request closed without merging**